collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -736,6 +736,7 @@ public void run() {
                 String bearing = intent.getStringExtra(BEARING_RESULT);
                 ((ODKView) mCurrentView).setBinaryData(bearing);
                 saveAnswersForCurrentScreen(DO_NOT_EVALUATE_CONSTRAINTS);
+                break;
```

Comment:
@shreyasnbhat What was the issue that you were facing? Adding a break statement here won't have any effect on the flow of the app.